BrowserKit/Sources/ToolbarKit/LocationView.swift

```diff
@@ -143,7 +165,20 @@ class LocationView: UIView, UITextFieldDelegate, ThemeApplicable {
     }
 
     private func setupLayout() {
-        addSubviews(urlTextField, gradientView, clearButton)
+        addSubviews(urlTextField, gradientView, clearButton, searchEngineButton)
+        searchEngineButton.addSubview(searchEngineImageView)
```

Comment:
We should add the image through `UIButton.Configuration` and add `contentInsets` instead of using a subview and giving the subview a size. However that would mean that the button cannot have a size of 40x40 or else we can't have the corner radius. The subview can't be used either way as it results in not haven't a button if you look at the view hierarchy.